core/TIBCO.Software.changes.txt

```diff
@@ -1,6 +1,6 @@
 
 TIBCO Software Inc. has made modifications to certain files of the Original Code of this component as further described below.
-The Original Code is “iText, a free JAVA-PDF library”, by Initial Developer, Bruno Lowagie, and Co-Developer, Paulo Soares.
+The Original Code is “iText, a free JAVA-PDF library�?, by Initial Developer, Bruno Lowagie, and Co-Developer, Paulo Soares.
```

Comment:
is that broken in the original update?

Reply:
I think it got broken during merge. I will fix it with simple quotes.